db/migrate/20201130142541_change_notifications_guid_not_null.rb

```diff
@@ -0,0 +1,16 @@
+
+class ChangeNotificationsGuidNotNull < ActiveRecord::Migration[5.2]
```

Comment:
Layout/LeadingEmptyLines: Unnecessary blank line at the beginning of the source.